### ISSUE TITLE
Questions about data for for flow loss training

### ISSUE THREAD

**Issue by JyunYuLai** [irfanICMLL/ETC-Real-time-Per-frame-Semantic-video-segmentation#11]
Hi,
Thank you for sharing the great work.
I have two questions related to the data.
1. As mentioned in the paper, training triplet is used for training, and I found this part implemented in dataset.py “VideoLongDatA”. But in the training code provided, “VideoData” is used for training which only provide pair data for training. I’m curious about what is the influence of these two different impl on the training results. Is this just simply a trade-off between training time and accuracy?
2. I’m curious about the“frame gap” parameter on the results. In the config provided, 3 is the default setting. Any study about this parameter?

I’m looking forward to your reply. 
Thank you.
